AutoRest/Modelers/Swagger/JsonConverters/PathItemRefConverter.cs

```diff
@@ -25,6 +25,13 @@ public override bool CanConvert(System.Type objectType)
         public override object ReadJson(JsonReader reader, System.Type objectType, object existingValue,
             JsonSerializer serializer)
         {
+            // is the leaf an vendor extension? "x-*..."
+            if (reader == null || reader.Path.Substring(reader.Path.LastIndexOf(".", StringComparison.Ordinal) + 1).StartsWith("x-",StringComparison.CurrentCulture))
+            {
+                // skip x-* vendor extensions when used where the path would be.
+                return new Dictionary < string, Operation >();
```

Comment:
Minor: spacing? I think these spaces will be removed the next time someone formats the document